gprofiler/utils/node.py

```diff
@@ -0,0 +1,243 @@
+import json
```

Comment:
This file is under `utils/` but logically it's like the rest of the profilers under `gprofiler/profilers/`, let's move as `gprofiler/profilers/node.py`?